### ISSUE TITLE
feature request :  a notification on sucsuss or error when adding key

### ISSUE THREAD

**Issue by zakidze** [vordgi/nimpl-inio#85]
i want a notification when a value is succsufully added a key or not 

like feedback when a value is succsusfly added

**Comment by vordgi** [comment]
Had you any moments when changes were not added?

**Comment by vordgi** [comment]
Inio update values in realtime. _Like here after 0:15_ - https://x.com/i/status/1803161947153498461

**Comment by zakidze** [comment]
yes  i had i added  value than collapse....then when i refrched a. value was not aded

**Comment by vordgi** [comment]
Okey, thanks!

I will try to find this case and notify about problems

**Comment by vordgi** [comment]
If you can find how to reproduce that case - it would be great!

**Comment by zakidze** [comment]
yes of course i would include this case to reproduce

**Comment by zakidze** [comment]
the next time i encounter i teell u
